### PR TITLE
CAMEL-8351 - simple implementation for breadcrumbId to be set as camel exhange's header

### PR DESCRIPTION
**Pull request by onders86**
simple implementation, if mimeheaders may have multiple values as string array for breadcrumbs header, breadcrumb generation strategy may need to be implemented and this may require a config uri param on the endpoint to inject such strategy

### CODE REVIEW

File: components/camel-spring-ws/src/main/java/org/apache/camel/component/spring/ws/SpringWebserviceConsumer.java [hunk 1]
```diff
@@ -74,6 +80,27 @@ public void invoke(MessageContext messageContext) throws Exception {
         }
 
     }
+    
+    private void populateExchangeWithBreadcrumbFromMessageContext(MessageContext messageContext, Exchange exchange) {
+        SaajSoapMessage saajSoap = (SaajSoapMessage) messageContext.getRequest();
+        SOAPMessage soapMesssageRequest = null;
+        if (saajSoap != null) {
+            soapMesssageRequest = saajSoap.getSaajMessage();
```
Comment (aldettinger) [inline components/camel-spring-ws/src/main/java/org/apache/camel/component/spring/ws/SpringWebserviceConsumer.java]: Spotted a simple typo there:
soapMesssageRequest -> soapMessageRequest

File: components/camel-spring-ws/src/test/java/org/apache/camel/component/spring/ws/ConsumerBreadcrumbIdTest.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+          
+                // request webservice
+                from("direct:webservice-marshall-asin")
+                        .marshal(jaxb)
+                        .to("spring-ws:http://localhost/?soapAction=http://www.stockquotes.edu/GetQuoteAsIn&webServiceTemplate=#webServiceTemplate")
+                        .convertBodyTo(String.class);
+                
+                // provide web service
+                from("spring-ws:soapaction:http://www.stockquotes.edu/GetQuoteAsIn?endpointMapping=#endpointMapping").setHeader("setin", constant("true"))
+                                                                                                                         .process(new StockQuoteResponseProcessor());
```
Comment (davsclaus) [inline components/camel-spring-ws/src/test/java/org/apache/camel/component/spring/ws/ConsumerBreadcrumbIdTest.java]: Can you remove all these empty lines

File: components/camel-spring-ws/src/main/java/org/apache/camel/component/spring/ws/SpringWebserviceConsumer.java (hunk 1 above)
Comment (davsclaus) [inline components/camel-spring-ws/src/main/java/org/apache/camel/component/spring/ws/SpringWebserviceConsumer.java]: well spotted. @onders86 can you fix this

**Comment by davsclaus** [comment]
@onders86 let us know if you get time to fix the minor comments on this PR so we can get the code merged

**Comment by onders86** [comment]
@davsclaus, i can fix the reviews by monday. i am very away from any computer. but i ll do as soon as i can. i know release getting closer. i hope fix goes with it

**Comment by davsclaus** [comment]
Thanks for the PR it has been merged. Do you mind closing this?

**Comment by onders86** [comment]
no problem. thanks

**Pull request closed without merging**